src/main/java/com/dreammaster/gthandler/recipes/SpaceAssemblerRecipes.java

```diff
@@ -44,6 +46,56 @@ public void run() {
                     ? FluidRegistry.getFluid("molten.indalloy140")
                     : FluidRegistry.getFluid("molten.solderingalloy");
 
+            IG_RecipeAdder.addSpaceAssemblerRecipe(
+                    new ItemStack[] { ItemList.Emitter_UHV.get(1),
+                            GTModHandler.getModItem(IndustrialCraft2.ID, "blockGenerator", 16, 5),
+                            GTModHandler.getModItem(IndustrialCraft2.ID, "blockGenerator", 16, 5),
+                            ItemList.Emitter_UHV.get(1),
+                            GTOreDictUnificator.get(OrePrefixes.plate, MaterialsUEVplus.ProtoHalkonite, 8),
+                            ItemList.EnergisedTesseract.get(1), ItemList.EnergisedTesseract.get(1),
+                            GTOreDictUnificator.get(OrePrefixes.plate, MaterialsUEVplus.ProtoHalkonite, 8),
+                            GTOreDictUnificator.get(OrePrefixes.plate, MaterialsUEVplus.ProtoHalkonite, 8),
+                            ItemList.EnergisedTesseract.get(1), ItemList.EnergisedTesseract.get(1),
+                            GTOreDictUnificator.get(OrePrefixes.plate, MaterialsUEVplus.ProtoHalkonite, 8),
+                            ItemList.Emitter_UHV.get(1),
+                            GTModHandler.getModItem(IndustrialCraft2.ID, "blockGenerator", 16, 5),
+                            GTModHandler.getModItem(IndustrialCraft2.ID, "blockGenerator", 16, 5),
+                            ItemList.Emitter_UHV.get(1), },
+                    new FluidStack[] { Materials.Plutonium.getMolten(512 * 144),
+                            GGMaterial.atomicSeparationCatalyst.getMolten(512 * 144),
+                            Materials.Tartarite.getMolten(512 * 144),
```

Comment:
The tartarite here is probably too expensive for something meant to be consumed

Reply:
Cut it to 64 at NotAPenguin's request